frontend/src/app/task-area/task-area.component.html

```diff
@@ -1,14 +1,22 @@
 <div class="task-area">
-  <h1>All Tasks</h1>
-  <input class="new-input" type="text" placeholder="Add a new task insdie ‘All’ category" />
+  <h1>{{ currentTab }} Tasks</h1>
+
+  <input (keyup.enter)="createTask()" [formControl]="taskText" class="new-input" type="text"
+    placeholder="Add a new task insdie '{{ currentTab }}' category" [ngClass]="{
+      invalid: taskText.errors && (taskText.dirty || taskText.touched),
+      valid: !taskText.errors && (taskText.dirty || taskText.touched)
+    }" />
+  <label class="error-message" *ngIf="taskText.invalid && taskText.value">This field invalid</label>
```

Comment:
For me better to read, when *ngIf and *ngFor directives in the start of attributes declarations.

Reply:
done

Reply:
I don't understand what is done xD